easybuild/easyconfigs/o/OpenSSL/OpenSSL-3.0.eb

```diff
@@ -0,0 +1,40 @@
+easyblock = 'EB_OpenSSL_wrapper'
+
+name = 'OpenSSL'
+version = '3.0'
```

Comment:
I think we should just have `3` here as they will start following semver going foward.